### ISSUE TITLE
`imagetools-core` 0.1.0-next.8 uses `||=` which is only available in Node 15

### ISSUE THREAD

**Issue by GrygrFlzr** [JonasKruckenberg/vite-imagetools#51]
Currently, `imagetools-core` 0.1.0-next.8 uses `||=` which is [only available in Node 15](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment#browser_compatibility).
This makes `vite-imagetools` unusable below Node 15 (can you even shim a Vite plugin with Babel?)

```node
C:\Users\GrygrFlzr\Documents\projects\sk-64b\node_modules\.pnpm\imagetools-core@0.1.0-next.8\node_modules\imagetools-core\dist\index.cjs:30
    blur ||= config.blur === 'true';
         ^^^

SyntaxError: Unexpected token '||='
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\GrygrFlzr\Documents\projects\sk-64b\node_modules\.pnpm\vite-imagetools@3.3.1\node_modules\vite-imagetools\dist\index.cjs:3:22)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
```

Tested on:
- Node v14.16.0
- pnpm v5.18.9
- Vite 2.1.4

**Comment by JonasKruckenberg** [comment]
Thanks for the heads up! Typescript should now compile down to es2018, I also fixed the github workflow so unit tests get run  in all nodes LTS versions 👍

**Comment by JonasKruckenberg** [comment]
Should be addressed in release `vite-imagetools@3.3.2-alpha.0`(`imagetools-core@0.1.0-next.9`)
(Don't ask me why it's alpha instead of next, I sometimes don't get Lerna 😄)
